crates/astria-sequencer-relayer/tests/blackbox/helper.rs

```diff
@@ -231,19 +230,31 @@ impl TestSequencerRelayer {
     }
 
     pub fn mount_block_response<const RELAY_SELF: bool>(&mut self, height: u32) -> BlockGuard {
+        use astria_core::primitive::v1::RollupId;
+        use rand::Rng;
+
+        let mut rng = rand::thread_rng();
```

Comment:
Tests should ideally not have random components because that can become very confusing when debugging. Can you just replace this `block_hash` with a non-random `[u8; 32]`?